tests/hcaptcha.spec.js

```diff
@@ -120,6 +126,27 @@ describe("hCaptcha", () => {
         expect(window.hcaptcha.remove.mock.calls[0][0]).toBe(MOCK_WIDGET_ID);
     });
 
+    it("can get Response", () => {
+        expect(window.hcaptcha.getResponse.mock.calls.length).toBe(0);
+        instance.getResponse();
```

Comment:
Lacking assert for the return value:
```js
const res = instance.getResponse();
expect(res).toBe(MOCK_TOKEN);
```

Reply:
Done https://github.com/hCaptcha/react-hcaptcha/pull/122/commits/09a2cab4b5e9dae2f5543b4e5839100ad0618c5a